confChamber.py

```diff
@@ -158,10 +144,10 @@
                     if not (vfatBoard.parentOH.parentAMC.fwVersion > 2):
                         vfatBoard.writeVFAT(int(event.vfatN), "ContReg3", int(event.trimRange),options.debug)
             
-            #print 'Comparing Curently Stored VFAT Registers with %s'%options.vfatConfig
-            #if options.vt1bump != 0:
-            #    print "Mismatches between write & readback valus for VThreshold1 should be exactly %i" %(options.vt1bump)
-            #readBackCheck(vfatTree, dict_readBack, ohboard, options.gtx)
+            if vfatBoard.parentOH.parentAMC.fwVersion > 2:
```

Comment:
this looks like independent of "compare" option. Why?

Reply:
The script will always perform the comparison, independent of if the `--compare` option is supplied.  It will only write the registers if the comparison option is *not* supplied.  This maintains the same functionality of the v2b electronics, see for example:

https://github.com/cms-gem-daq-project/vfatqc-python-scripts/blob/a5747b11e5f71a104a71c28132f97aef2996f2e2/confChamber.py#L119-L130

Reply:
ah, ok